### ISSUE TITLE
setup documentation

### ISSUE THREAD

**Issue by ren-justb** [brenkmanco/hackpdm#57]
Would it be possible to get setup documentation? I've successfully built the program but cannot get past the profile manager window without a pwa directory error

Thanks!

**Comment by matt454357** [comment]
Yeah, there's a simple trick to get past that little bug. I can't remember it right now. I can post it next week when I'm back from a trip.

**Comment by ren-justb** [comment]
Thanks!

**Comment by matt454357** [comment]
I entered the following data, and the program started as expected:
![image](https://user-images.githubusercontent.com/7604932/192594201-0a411b79-f04e-4efd-9ba4-47d488905825.png)

I assume you are getting an error box with "Still can't get a profile. Can't find the local pwa directory."
![image](https://user-images.githubusercontent.com/7604932/192594761-8889eb35-7016-40a1-8fb8-f5854f57bf53.png)

If you post your config file, I can probably help.  You should be able to find it somewhere around here:
```C:\Users\<your user>\AppData\Local\HackPDM\HackPDM.exe_Url_<build ref hex string>\1.0.<other numbers>\user.config```

**Comment by ren-justb** [comment]
Thanks! yes that is the error I am getting, here is the text from my config file:

![image](https://user-images.githubusercontent.com/111295743/192815362-1bf5f3e0-f512-471a-b1e4-249bff283470.png)

When I try rerunning the setup app it doesn't seem to overwrite this, I may need to uninstall/purge and reinstall

**Comment by matt454357** [comment]
Here's a setup guide:

https://github.com/brenkmanco/hackpdm/wiki/Setup

I'm happy to elaborate, if you still have questions.
